test/integration/manifests/cni/cni-installer-v1-windows.yaml

```diff
@@ -36,19 +36,39 @@ spec:
       hostNetwork: true
       serviceAccountName: azure-cni
       tolerations:
-      - operator: Exists
-        effect: NoSchedule
-      - key: CriticalAddonsOnly
-        operator: Exists
-      - effect: NoExecute
-        operator: Exists
-      containers:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - operator: "Exists"
+          effect: NoExecute
+        - operator: "Exists"
+          effect: NoSchedule
+      initContainers:
+        - name: delete-azure-vnet-telemetry
+          image: mcr.microsoft.com/powershell:lts-nanoserver-ltsc2022
+          command: ["pwsh", "-command"]
+          args: ["if (Get-Process -Name 'azure-vnet-telemetry' -ErrorAction SilentlyContinue) { Stop-Process -Name 'azure-vnet-telemetry' -Force }"]
         - name: cni-drop
-          image: ${CNI_IMAGE}
+          image: ${DROP_GZ_URL}
           imagePullPolicy: Always
+          command: ["%CONTAINER_SANDBOX_MOUNT_POINT%/dropgz.exe"]
+          args:
+            - deploy
+            - azure-vnet.exe
+            - -o
+            - /k/azurecni/bin/azure-vnet.exe
+            - azure-vnet-ipam.exe
+            - -o
+            - /k/azurecni/bin/azure-vnet-ipam.exe
+            - azure-vnet-telemetry.exe
+            - -o
+            - /k/azurecni/bin/azure-vnet-telemetry.exe

```

Comment:
have to copy telemetry.config file as well

Reply:
Yeah, still testing the init container part, will ping once that is verified.